lib/mayaUsd/ufe/Global.cpp

```diff
@@ -120,6 +118,10 @@ void mayaStopWaitCursor() { MGlobal::executeCommand("waitCursor -state 0"); }
 // Note: MayaUsd::ufe::getStage takes two parameters, so wrap it in a function taking only one.
 PXR_NS::UsdStageWeakPtr mayaGetStage(const Ufe::Path& path) { return MayaUsd::ufe::getStage(path); }
 
+void displayInfoMessage(const std::string& msg) { MGlobal::displayInfo(msg.c_str()); }
```

Comment:
is this going to the maya listener? (the thing showing the red text bottom right)

Reply:
Yes except being info it won't be red

Reply:
This prints to the Maya script editor and the command line. Three types of message: info, warning & error (same as USD).